script.js

```diff
@@ -122,6 +204,367 @@ const introData = intro.map((item) => {
+  });
+  const btn = document.createElement("button");
+  btn.innerHTML = elem.btn;
+  div.appendChild(btn);
+  return div;
+});
+const cardData3 = cards.map((elem) => {
+  const div = document.createElement("div");
+  div.className = elem.cls2;
+  const h3 = document.createElement("h3");
+  h3.innerHTML = elem.h3;
+  div.appendChild(h3);
+  const para = document.createElement("p");
+  para.innerHTML = elem.para;
+  div.appendChild(para);
+  const ul = document.createElement("ul");
+  elem.list.map((item) => {
+    const li = document.createElement("li");
+    const aTag = document.createElement("a");
+    aTag.innerHTML = item;
+    li.appendChild(aTag);
+    ul.appendChild(li);
+    div.appendChild(ul);
+  });
+  const btn = document.createElement("button");
+  btn.innerHTML = elem.btn;
+  div.appendChild(btn);
+  return div;
+});
+const cardData4 = cards.map((elem) => {
+  const div = document.createElement("div");
+  div.className = elem.cls3;
+  const h3 = document.createElement("h3");
+  h3.innerHTML = elem.h3;
+  div.appendChild(h3);
+  const para = document.createElement("p");
+  para.innerHTML = elem.para;
+  div.appendChild(para);
+  const ul = document.createElement("ul");
+  elem.list.map((item) => {
+    const li = document.createElement("li");
+    const aTag = document.createElement("a");
+    aTag.innerHTML = item;
+    li.appendChild(aTag);
+    ul.appendChild(li);
+    div.appendChild(ul);
+  });
+  const btn = document.createElement("button");
+  btn.innerHTML = elem.btn;
+  div.appendChild(btn);
+  return div;
+});
+const cardData5 = cards.map((elem) => {
+  const div = document.createElement("div");
+  div.className = elem.cls4;
+  const h3 = document.createElement("h3");
+  h3.innerHTML = elem.h3;
+  div.appendChild(h3);
+  const para = document.createElement("p");
+  para.innerHTML = elem.para;
+  div.appendChild(para);
+  const ul = document.createElement("ul");
+  elem.list.map((item) => {
+    const li = document.createElement("li");
+    const aTag = document.createElement("a");
+    aTag.innerHTML = item;
+    li.appendChild(aTag);
+    ul.appendChild(li);
+    div.appendChild(ul);
+  });
+  const btn = document.createElement("button");
+  btn.innerHTML = elem.btn;
+  div.appendChild(btn);
+  return div;
+});
+
+const cardData6 = cards.map((elem) => {
+  const div = document.createElement("div");
+  div.className = elem.cls5;
+  const h3 = document.createElement("h3");
+  h3.innerHTML = elem.h3;
+  div.appendChild(h3);
+  const para = document.createElement("p");
+  para.innerHTML = elem.para;
+  div.appendChild(para);
+  const ul = document.createElement("ul");
+  elem.list.map((item) => {
+    const li = document.createElement("li");
+    const aTag = document.createElement("a");
+    aTag.innerHTML = item;
+    li.appendChild(aTag);
+    ul.appendChild(li);
+    div.appendChild(ul);
+  });
+  const btn = document.createElement("button");
+  btn.innerHTML = elem.btn;
+  div.appendChild(btn);
+  return div;
+});
+

```

Comment:
- [ ] @team great work in your projects section, however, you were supposed to create a looping method to generate for you the sis card instead of creating each card individually this will avid code repetition nd also reduce the size of your code leading to better readability.

Reply:
- [ ] @team kindly check on your desktop design, I cannot see the project button, and also the button in the mobile view doesn't work am unable to open the projects popup. Kindly ensure the design matches the design given,  below is a comparison.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/202844741-b22a0100-0711-4857-b68f-b4258051b210.png) |  ![image](https://user-images.githubusercontent.com/61408860/202844783-79776b95-00e4-4f8e-9923-78749087861c.png)